src/enums/ClaimConditionType.ts

```diff
@@ -0,0 +1,27 @@
+enum ClaimConditionType {
+  StartTime,
+  LimitPerWallet,
+  MerkleRoot,
+  WaitTimeBetweenClaims,
+}
+
+/**
+ * Questions:
+ *
+ * 1. Can there only be a single `startTimestamp` condition? What is a `startTimestamp` condition?
+ *
+ * 2. What is `maxMintSupply`?
+ *
+ * 3. What is `currentMintSupply`?
```

Comment:
always 0, client should not be setting this. internal to the contract.

Reply:
we should include this as a comment in the contract